src/spsc/mod.rs

```diff
@@ -394,6 +394,30 @@ macro_rules! impl_ {
                 }
             }
         }
+
+        impl<T, N, C> Clone for Queue<T, N, $uxx, C>
+        where
+            T: Clone,
+            N: ArrayLength<T>,
+            C: sealed::XCore,
+        {
+            fn clone(&self) -> Self {
+                let mut new = Queue {
```

Comment:
I think you'll need to specify the full type here (`let mut new: Queue<..> = ..`), otherwise method lookup will fail below.

Reply:
Yes, this does indeed solve the problem 👍